frontend/utility/OBSSparkle.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#import <QObject>
```

Comment:
That is correct because the header is used exclusively in an ObjC++ file.

Reply:
Got it.

Reply:
I've looked into this again, and we might just be lucky here:

That specific header will also be used by `moc` to create an associated `cpp` file, which will be compiled as plain C++ (hence why some parts are guarded with a check for the `__OBJC__` preprocessor definition).

However as the compiler used will still be `AppleClang`, it will understand the `import` statement used in the generated file (which itself just includes this header) and thus `QObject` is correctly imported.

As the file itself is only used on macOS, this seems to be fine.